src/couchapps/WMStats/_attachments/index.html

```diff
@@ -4,6 +4,9 @@
   <head>
     <title>WMStats</title>
     <style type="text/css" title="currentStyle">
+        @import "css/logdb.css";
+        @import "css/kube.min.css";
```

Comment:
Seangchan,
sorry for my silence, I just got to Geneva. I'm attaching required files. 
Valentin.
On  0, ticoann <notifications@github.com> wrote:
> > @@ -4,6 +4,9 @@
> >    <head>
> >      <title>WMStats</title>
> >      <style type="text/css" title="currentStyle">
> > +        @import "css/logdb.css";
> > +        @import "css/kube.min.css";
> 
> Hi Valentin, This css file is missing from the head or patch? could you update that one?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMCore/pull/6016/files#r33416760

*,*:before,*:after{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-family:sans-serif;-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}html,body,div,span,object,iframe,p,blockquote,pre,a,abbr,acronym,address,big,cite,code,del,dfn,em,img,ins,kbd,q,s,samp,small,strike,strong,sub,sup,tt,var,b,u,i,center,dl,dt,dd,ol,ul,li,fieldset,form,label,legend,table,caption,tbody,tfoot,thead,tr,th,td,article,aside,canvas,details,embed,figure,figcaption,footer,header,hgroup,menu,nav,output,ruby,section,summary,time,mark,audio,video,h1,h2,h3,h4,h5,h6{margin:0;padding:0;border:0;outline:0;font-size:100%;vertical-align:baseline;background:transparent}a:active,a:hover{outline:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,nav,section,embed,object,iframe,audio,video,canvas,progress,meter,output,textarea{display:block}audio:not([controls]){display:none;height:0}blockquote,q{quotes:none}blockquote p:before,blockquote p:after,q:before,q:after{content:'';content:none}table{border-collapse:collapse;border-spacing:0}caption,th,td{text-align:left;vertical-align:top}thead th,thead td{font-weight:bold;vertical-align:bottom}a img,th img,td img{vertical-align:top}button,input,select,textarea{margin:0}textarea{overflow:auto;vertical-align:top;resize:vertical}button{width:auto;overflow:visible}input[type="reset"],input[type="submit"],input[type="file"],input[type="radio"],input[type="checkbox"],select,button{cursor:pointer}input[type="radio"],input[type="checkbox"]{font-size:110%;position:relative;top:-1px;margin-right:3px}input[type="search"]{-webkit-appearance:textfield}input[type="search"]::-webkit-search-decoration,input[type="search"]::-webkit-search-cancel-button{-webkit-appearance:none}hr{display:block;height:1px;border:0;border-top:1px solid #ddd}img,video,audio,embed,object{max-width:100%}img,video,embed,object{height:auto}embed,object{height:100%}img{vertical-align:middle;-ms-interpolation-mode:bicubic}body{font-family:"Helvetica Neue",Helvetica,Tahoma,sans-serif;font-size:.9375em;line-height:1.65em;background:#fff;color:#222}a{color:#134da5}a:focus,a:hover{color:#de2c3b}h1,h2,h3,h4,h5,h6,.h1,.h2,.h3,.h4,.h5,.h6{font-family:"Helvetica Neue",Helvetica,Tahoma,sans-serif;font-weight:bold;color:#000;text-rendering:optimizeLegibility;margin:0 0 .5em 0}h1,.h1,h2,.h2{line-height:1.1}h3,.h3,h4,.h4{line-height:1.3}h1,.h1{font-size:2.25em}h2,.h2{font-size:1.5em;margin-bottom:.7em}h3,.h3{font-size:1.3125em}h4,.h4{font-size:1.125em}h5,.h5{font-size:1em}h6,.h6{font-size:.75em;text-transform:uppercase}.lead{font-size:1.3125em;line-height:1.5;margin-bottom:1.0999999999999999em}p,ul,ol,dl,dd,dt,blockquote,td,th{line-height:1.65em}ul,ol,ul ul,ol ol,ul ol,ol ul{margin:0 0 0 3.3em}ul li,ol li{text-align:left}ol ol li{list-style-type:lower-alpha}ol ol ol li{list-style-type:lower-roman}p,ul,ol,dl,blockquote,hr,pre,table,form,fieldset,figure,address{margin-bottom:1.65em}blockquote{position:relative;font-style:italic;font-size:1.125em;margin-left:2.4749999999999996em;padding-left:1.65em;border-left:2px solid #ddd}blockquote p{margin-bottom:.5em}blockquote small,cite{color:rgba(0,0,0,0.4);font-style:italic}small,blockquote cite{font-size:12.75px;line-height:1}address{font-style:normal}dl dt{font-weight:bold}dd{margin-left:1.65em}s,del{text-decoration:line-through}abbr[title],dfn[title]{border-bottom:1px dotted #0f0f0f;cursor:help}strong,b{font-weight:bold}em,i{font-style:italic}sub,sup{font-size:11.25px;line-height:0;position:relative}sup{top:-0.5em}sub{bottom:-0.25em}figcaption{margin:.3em 0;font-size:12.75px;font-style:italic}ins,u{text-decoration:underline}mark{background-color:#ffc800;color:#0f0f0f;text-decoration:none}pre,code,kbd,samp,var,output{font-size:90%;font-style:normal;font-family:Menlo,Monaco,Consolas,"Courier New",monospace}pre{margin-top:1.65em;font-size:100%;line-height:1.5;color:#222;overflow:auto}code,samp,kbd{padding:3px 6px 2px 6px;display:inline-block;line-height:1;border-radius:2px}code{background:#f4f4f4;border:1px solid #d4d4d4}pre code{font-size:100%;border:0;padding:0;background:0;line-height:1.65em}var{color:rgba(0,0,0,0.5)}samp{background:#d3e3fb;border:1px solid #b2cef8}kbd{background:#0f0f0f;color:rgba(255,255,255,0.85);white-space:nowrap}button:active,button:focus{outline:0}textarea,select{font-family:"Helvetica Neue",Helvetica,Tahoma,sans-serif;font-size:1em;box-shadow:none}textarea,select[multiple],select[multiple="multiple"]{padding:.3em .35em;line-height:1.35em;width:100%}input[type="text"],input[type="password"],input[type="email"],input[type="url"],input[type="phone"],input[type="tel"],input[type="number"],input[type="datetime"],input[type="date"],input[type="month"],input[type="color"],input[type="time"],input[type="search"],input[type="datetime-local"]{font-family:"Helvetica Neue",Helvetica,Tahoma,sans-serif;font-size:1em;box-shadow:none;padding:.3em .35em;line-height:1.65em;border-radius:0;outline:0}select[multiple],select[multiple="multiple"],textarea,input[type="text"],input[type="password"],input[type="email"],input[type="url"],input[type="phone"],input[type="tel"],input[type="number"],input[type="datetime"],input[type="date"],input[type="month"],input[type="color"],input[type="time"],input[type="search"],input[type="datetime-local"]{background:#fff;border:1px solid #ccc;position:relative;z-index:2;-webkit-appearance:none}textarea[disabled],input[type="text"][disabled],input[type="password"][disabled],input[type="email"][disabled],input[type="url"][disabled],input[type="phone"][disabled],input[type="tel"][disabled],input[type="number"][disabled],input[type="datetime"][disabled],input[type="date"][disabled],input[type="month"][disabled],input[type="color"][disabled],input[type="time"][disabled],input[type="search"][disabled],input[type="datetime-local"][disabled]{resize:none;color:rgba(0,0,0,0.5)}select[disabled],input[type="checkbox"][disabled],input[type="radio"][disabled]{cursor:default}input::-moz-focus-inner,button::-moz-focus-inner{border:0;padding:0}input[type="range"]{position:relative;top:3px}select{margin-bottom:0!important}fieldset{padding:1.65em;margin-bottom:1.65em;border:1px solid #ccc}fieldset *:last-child{margin-bottom:0!important}legend{font-weight:bold;padding:0 1em;margin-left:-1em}table{max-width:100%;width:100%;empty-cells:show}table caption{text-transform:uppercase;padding:0 1.0999999999999999em;color:rgba(0,0,0,0.4);font-size:12.75px}table th,table td{border-bottom:1px solid #eee;padding:.825em 1.0999999999999999em}table tfoot th,table tfoot td{color:rgba(0,0,0,0.4)}.units-container:after,.units-row:after{content:"";display:table;clear:both}.units-container:after,.units-row:after{content:"";display:table;clear:both}.units-container{padding-top:1px;margin-top:-1px}.units-row{margin-bottom:1.65em}.width-100,.unit-100{width:100%}.width-90,.unit-90{width:90%}.width-80,.unit-80{width:80%}.width-75,.unit-75{width:75%}.width-70,.unit-70{width:70%}.width-66,.unit-66{width:66.6%}.width-65,.unit-65{width:65%}.width-60,.unit-60{width:60%}.width-50,.unit-50{width:50%}.width-40,.unit-40{width:40%}.width-35,.unit-35{width:35%}.width-33,.unit-33{width:33.3%}.width-30,.unit-30{width:30%}.width-25,.unit-25{width:25%}.width-20,.unit-20{width:20%}.width-10,.unit-10{width:10%}.units-row .unit-90,.units-row .unit-80,.units-row .unit-75,.units-row .unit-70,.units-row .unit-66,.units-row .unit-65,.units-row .unit-60,.units-row .unit-50,.units-row .unit-40,.units-row .unit-35,.units-row .unit-33,.units-row .unit-30,.units-row .unit-25,.units-row .unit-20,.units-row .unit-10{float:left;margin-left:3%}.units-row .unit-90:first-child,.units-row .unit-80:first-child,.units-row .unit-75:first-child,.units-row .unit-70:first-child,.units-row .unit-66:first-child,.units-row .unit-65:first-child,.units-row .unit-60:first-child,.units-row .unit-50:first-child,.units-row .unit-40:first-child,.units-row .unit-35:first-child,.units-row .unit-33:first-child,.units-row .unit-30:first-child,.units-row .unit-25:first-child,.units-row .unit-20:first-child,.units-row .unit-10:first-child{margin-left:0}.units-row .unit-90{width:89.7%}.units-row .unit-80{width:79.4%}.units-row .unit-75{width:74.25%}.units-row .unit-70{width:69.1%}.units-row .unit-66{width:65.66666666666666%}.units-row .unit-65{width:65.66666666666666%}.units-row .unit-60{width:58.800000000000004%}.units-row .unit-50{width:48.5%}.units-row .unit-40{width:38.2%}.units-row .unit-35{width:31.333333333333332%}.units-row .unit-33{width:31.333333333333332%}.units-row .unit-30{width:27.9%}.units-row .unit-25{width:22.75%}.units-row .unit-20{width:17.6%}.units-row .unit-10{width:7.3%}.unit-push-90,.unit-push-80,.unit-push-75,.unit-push-70,.unit-push-66,.unit-push-65,.unit-push-60,.unit-push-50,.unit-push-40,.unit-push-35,.unit-push-33,.unit-push-30,.unit-push-25,.unit-push-20,.unit-push-10{position:relative}.unit-push-90{left:92.7%}.unit-push-80{left:82.4%}.unit-push-75{left:77.25%}.unit-push-70{left:72.1%}.unit-push-66{left:68.66666666666666%}.unit-push-65{left:68.66666666666666%}.unit-push-60{left:61.800000000000004%}.unit-push-50{left:51.5%}.unit-push-40{left:41.2%}.unit-push-35{left:34.33333333333333%}.unit-push-33{left:34.33333333333333%}.unit-push-30{left:30.9%}.unit-push-25{left:25.75%}.unit-push-20{left:20.6%}.unit-push-10{left:10.3%}.units-row .unit-push-right{float:right}.units-row .unit-role-right{margin-left:3%;float:right}.units-row .unit-role-left{margin-left:0}.centered,.unit-centered{float:none!important;margin:0 auto!important}.unit-padding{padding:1.65em}.units-padding .unit-100,.units-padding .unit-90,.units-padding .unit-80,.units-padding .unit-75,.units-padding .unit-70,.units-padding .unit-66,.units-padding .unit-65,.units-padding .unit-60,.units-padding .unit-50,.units-padding .unit-40,.units-padding .unit-35,.units-padding .unit-33,.units-padding .unit-30,.units-padding .unit-25,.units-padding .unit-20,.units-padding .unit-10{padding:1.65em}.units-split .unit-90,.units-split .unit-80,.units-split .unit-75,.units-split .unit-70,.units-split .unit-66,.units-split .unit-65,.units-split .unit-60,.units-split .unit-50,.units-split .unit-40,.units-split .unit-35,.units-split .unit-33,.units-split .unit-30,.units-split .unit-25,.units-split .unit-20,.units-split .unit-10{margin-left:0}.units-split .unit-90{width:90%}.units-split .unit-80{width:80%}.units-split .unit-75{width:75%}.units-split .unit-70{width:70%}.units-split .unit-66{width:66.6%}.units-split .unit-65{width:65%}.units-split .unit-60{width:60%}.units-split .unit-50{width:50%}.units-split .unit-40{width:40%}.units-split .unit-35{width:35%}.units-split .unit-33{width:33.3%}.units-split .unit-30{width:30%}.units-split .unit-25{width:25%}.units-split .unit-20{width:20%}.units-split .unit-10{width:10%}.blocks-2,.blocks-3,.blocks-4,.blocks-5,.blocks-6{padding-left:0;list-style:none;margin-left:-3%}.blocks-2:after,.blocks-3:after,.blocks-4:after,.blocks-5:after,.blocks-6:after{content:"";display:table;clear:both}.blocks-2:after,.blocks-3:after,.blocks-4:after,.blocks-5:after,.blocks-6:after{content:"";display:table;clear:both}.blocks-2>li,.blocks-3>li,.blocks-4>li,.blocks-5>li,.blocks-6>li{height:auto;float:left;margin-bottom:1.65em;margin-left:3%}.blocks-2>li ul,.blocks-3>li ul,.blocks-4>li ul,.blocks-5>li ul,.blocks-6>li ul{list-style-type:disc}.blocks-2>li ul ul,.blocks-3>li ul ul,.blocks-4>li ul ul,.blocks-5>li ul ul,.blocks-6>li ul ul{list-style-type:circle}.blocks-2>li li,.blocks-3>li li,.blocks-4>li li,.blocks-5>li li,.blocks-6>li li{float:none;margin:0}.blocks-2>li{width:47%}.blocks-3>li{width:30.333333333333332%}.blocks-4>li{width:22%}.blocks-5>li{width:17%}.blocks-6>li{width:13.666666666666666%}.block-first{clear:both}table.table-bordered td,table.table-bordered th{border:1px solid #eee}table.table-simple td,table.table-simple th,table.table-simple caption{border:0;padding-left:0}table.table-flat td,table.table-flat th,table.table-flat caption{border:0;padding:0}table.table-stroked td,table.table-stroked th{border-bottom:1px solid #eee}table.table-stripped tbody tr:nth-child(odd) td{background:#f8f8f8}table.table-hovered tbody tr:hover td{background-color:#f4f4f4}.table-container{width:100%;overflow:auto;margin-bottom:1.65em}.table-container table{margin-bottom:0}.table-container::-webkit-scrollbar{-webkit-appearance:none;width:14px;height:14px}.table-container::-webkit-scrollbar-thumb{border-radius:8px;border:3px solid #fff;background-color:rgba(0,0,0,0.3)}.forms .btn,.forms input[type="submit"].btn,.forms button,.forms input[type="submit"],.forms input[type="reset"]{padding:.3625em 1.3em}.forms .btn-outline{padding:.3em 1.3em}.forms .btn-outline.bold{padding:.2375em 1.3em}.forms label{display:block;margin-bottom:1.0999999999999999em}.forms input[type="text"],.forms input[type="password"],.forms input[type="email"],.forms input[type="url"],.forms input[type="phone"],.forms input[type="tel"],.forms input[type="number"],.forms input[type="datetime"],.forms input[type="date"],.forms input[type="month"],.forms input[type="color"],.forms input[type="time"],.forms input[type="search"],.forms input[type="range"],.forms input[type="file"],.forms input[type="datetime-local"],.forms textarea,.forms select{display:block}.forms-inline input[type="text"],.forms-inline-list input[type="text"],.forms-inline input[type="password"],.forms-inline-list input[type="password"],.forms-inline input[type="email"],.forms-inline-list input[type="email"],.forms-inline input[type="url"],.forms-inline-list input[type="url"],.forms-inline input[type="phone"],.forms-inline-list input[type="phone"],.forms-inline input[type="tel"],.forms-inline-list input[type="tel"],.forms-inline input[type="number"],.forms-inline-list input[type="number"],.forms-inline input[type="datetime"],.forms-inline-list input[type="datetime"],.forms-inline input[type="date"],.forms-inline-list input[type="date"],.forms-inline input[type="month"],.forms-inline-list input[type="month"],.forms-inline input[type="color"],.forms-inline-list input[type="color"],.forms-inline input[type="time"],.forms-inline-list input[type="time"],.forms-inline input[type="search"],.forms-inline-list input[type="search"],.forms-inline input[type="range"],.forms-inline-list input[type="range"],.forms-inline input[type="file"],.forms-inline-list input[type="file"],.forms-inline input[type="datetime-local"],.forms-inline-list input[type="datetime-local"],.forms-inline textarea,.forms-inline-list textarea,.forms-inline select,.forms-inline-list select{display:inline-block}.forms-list,.forms-inline,.forms-inline-list{margin:0;padding:0;margin-bottom:1.0999999999999999em;list-style:none}.forms-list label,.forms-inline label,.forms-inline-list li,.forms-inline-list li label{display:inline-block;margin-bottom:0}.forms-inline-list li label{margin-right:.825em}.forms-inline-list li{margin-bottom:3px}.forms-list li{margin-bottom:6px}.forms-desc{margin-top:4px;color:rgba(0,0,0,0.4);font-size:12.75px;line-height:1.4em}input[type="text"],input[type="password"],input[type="email"],input[type="url"],input[type="phone"],input[type="tel"],input[type="number"],input[type="datetime"],input[type="date"],input[type="month"],input[type="color"],input[type="time"],input[type="search"],input[type="datetime-local"],textarea{-moz-transition:border ease .5s;transition:border ease .5s}.error,.success{font-weight:normal;font-size:12.75px}input.input-error,textarea.input-error,select.input-error,.input-error{border-color:#de2c3b;box-shadow:0 0 0 2px rgba(222,44,59,0.3),0 1px 2px rgba(0,0,0,0.2) inset}input.input-success,textarea.input-success,select.input-success,.input-success{border-color:#2c9f42;box-shadow:0 0 0 2px rgba(44,159,66,0.3),0 1px 2px rgba(0,0,0,0.2) inset}input.input-gray,textarea.input-gray,select.input-gray,.input-gray{border-color:#b3b6b7;box-shadow:0 0 0 2px rgba(179,182,183,0.4),0 1px 2px rgba(0,0,0,0.2) inset}input[type="text"]:focus,input[type="password"]:focus,input[type="email"]:focus,input[type="url"]:focus,input[type="phone"]:focus,input[type="tel"]:focus,input[type="number"]:focus,input[type="datetime"]:focus,input[type="date"]:focus,input[type="month"]:focus,input[type="color"]:focus,input[type="time"]:focus,input[type="search"]:focus,input[type="datetime-local"]:focus,textarea:focus{outline:0;border-color:#2575ed;box-shadow:0 0 0 2px rgba(37,117,237,0.3),0 1px 2px rgba(0,0,0,0.2) inset}input.input-search,input[type="search"]{padding-right:.5em;padding-left:.5em;margin-bottom:0;border-radius:15px}input.input-on-black{border:1px solid rgba(255,255,255,0.1);background:rgba(255,255,255,0.35)}input.input-on-black::-webkit-input-placeholder{color:rgba(255,255,255,0.6)}input.input-on-black::-moz-placeholder{color:rgba(255,255,255,0.6)}input.input-on-black:focus,input.input-on-black.active{border:1px solid #fff;background:#fff;box-shadow:none}input.input-on-black:focus::-webkit-input-placeholder,input.input-on-black.active::-webkit-input-placeholder{color:#aaa}input.input-on-black:focus::-moz-placeholder,input.input-on-black.active::-moz-placeholder{color:#aaa}input.input-big{font-size:18px}input.input-small{font-size:12.75px}input.input-smaller{font-size:11.25px}.input-groups{display:table!important}.input-groups input{width:100%}.input-groups input,.input-groups .input-append,.input-groups .input-prepend,.input-groups .btn-append{display:table-cell!important}.input-groups .btn-append,.input-groups .input-append,.input-groups .input-prepend{width:1%;vertical-align:middle}.input-groups .input-append,.input-groups .input-prepend{background-color:#f4f4f4;border:1px solid #d4d4d4;margin:0;padding:.45em .75em .15em .75em;color:rgba(0,0,0,0.6);line-height:1.65em;font-size:12.75px;white-space:nowrap}.input-groups .input-prepend{border-right:0}.input-groups .input-append{position:relative;z-index:1;border-left:none}.input-groups .btn-append .btn{display:block;height:auto;border-radius:0 2px 2px 0}.navbar{font-size:105%;margin-bottom:1.65em}.navbar:after{content:"";display:table;clear:both}.navbar:after{content:"";display:table;clear:both}.navbar.navbar-left{float:left}.navbar.navbar-right{float:right}.navbar ul{list-style:none;margin:0}.navbar ul:after{content:"";display:table;clear:both}.navbar ul:after{content:"";display:table;clear:both}.navbar li{float:left;margin-right:1.65em}.navbar.navbar-right li{margin-right:0;margin-left:1.65em}.navbar a,.navbar span{display:block;text-decoration:none}.navbar a:hover{color:#de2c3b;text-decoration:underline}.navbar li.active a,.navbar span{text-decoration:none;cursor:text;color:rgba(0,0,0,0.4)}.navigation-toggle{display:none;text-transform:uppercase;position:relative;color:#0f0f0f;background-color:#fff}.navigation-toggle:after{position:absolute;z-index:1;top:50%;margin-top:-12px;left:10px;content:"\2630";font-size:24px;line-height:1}.navigation-toggle.navigation-toggle-black{background-color:#0f0f0f;color:#fff}.navigation-toggle.navigation-toggle-show{margin-bottom:1.65em}.navigation-toggle span{position:relative;z-index:2;cursor:pointer;display:block;padding:10px 20px 10px 40px}.navigation-fixed{position:fixed;top:0;left:0;z-index:101}.navbar-pills a,.navbar-pills span{padding:.825em 1.65em}.navbar-pills a:hover{color:#0f0f0f;background:#eee}.navbar-pills li.active a,.navbar-pills span{text-decoration:none;cursor:text;color:rgba(0,0,0,0.4);background:#eee}.navbar-pills li{margin-right:1px}.navbar-pills.navbar-right li{margin-left:1px}.fullwidth ul{width:100%}.fullwidth li{float:none!important;margin:0;display:table-cell;width:1%;text-align:center}.fullwidth li a,.fullwidth li span{display:block}.nav{margin-bottom:1.65em}.nav ul{list-style:none;margin:0}.nav ul li ul{margin-left:2em;font-size:.95em}.nav a,.nav span{display:block;padding:.5em 0}.nav a{color:#0f0f0f;text-decoration:none}.nav a:hover{color:#de2c3b;text-decoration:underline}.nav li.active a,.nav span{text-decoration:none;cursor:text;color:rgba(0,0,0,0.4)}.nav i.fa{width:1.65em}.nav-stacked li{margin-bottom:1px}.nav-stacked a,.nav-stacked span{padding:.6em .7em}.nav-stacked a{background:#f6f6f6}.nav-stacked a:hover{color:#000;background:#eee}.nav-stacked li.active a,.nav-stacked span{background:#fff;text-decoration:none;cursor:text;color:rgba(0,0,0,0.3)}.nav-stats li{position:relative}.nav-stats a,.nav-stats span{padding-right:50px}.nav-stats sup,.nav-stats .badge{position:absolute;top:50%;right:0}.nav-stats sup{color:rgba(0,0,0,0.4)}.nav-stats .badge{margin-top:-8px}.nav-tabs{border-bottom:1px solid #e3e3e3;margin-bottom:1.65em}.nav-tabs:after{content:"";display:table;clear:both}.nav-tabs:after{content:"";display:table;clear:both}.nav-tabs ul{list-style:none;margin:0}.nav-tabs li{float:left;margin-right:2px}.nav-tabs a,.nav-tabs span{display:block;line-height:1;padding:.825em 1.65em;border:1px solid transparent}.nav-tabs a{color:rgba(0,0,0,0.5);text-decoration:none}.nav-tabs a:focus,.nav-tabs a:hover{color:#0f0f0f;text-decoration:underline;background-color:#eee}.nav-tabs li.active a,.nav-tabs span{color:#0f0f0f;background:#fff;position:relative;border:1px solid #ddd;border-bottom:1px solid #fff;bottom:-1px;cursor:default;text-decoration:none}.breadcrumbs{margin-bottom:1.65em}.breadcrumbs:after{content:"";display:table;clear:both}.breadcrumbs:after{content:"";display:table;clear:both}.breadcrumbs ul{font-size:.9em;color:rgba(0,0,0,0.4);list-style:none;margin:0}.breadcrumbs ul:after{content:"";display:table;clear:both}.breadcrumbs ul:after{content:"";display:table;clear:both}.breadcrumbs li{float:left;margin-right:5px}.breadcrumbs li+li:before{content:" > ";color:#aaa;font-size:12px;margin:0 7px 0 5px;position:relative;top:-1px}.breadcrumbs.breadcrumbs-path li+li:before{content:" / ";top:0}.breadcrumbs a{color:#0f0f0f;text-decoration:none}.breadcrumbs a:hover{color:#0f0f0f;text-decoration:underline}.breadcrumbs li.active a,.breadcrumbs span,.breadcrumbs li.active a:hover{text-decoration:none;cursor:text;color:rgba(0,0,0,0.4)}.pagination{position:relative;left:-9px;margin-left:0;list-style:none}.pagination:after{content:"";display:table;clear:both}.pagination:after{content:"";display:table;clear:both}.pagination li{float:left;margin-right:2px}.pagination a,.pagination span{display:block;padding:7px 9px;line-height:1;border-radius:2px;color:#0f0f0f;text-decoration:none}.pagination span,.pagination li.active a,.pagination li.active a:hover{color:#fff;background-color:#0f0f0f;cursor:text}.pagination a:focus,.pagination a:hover{text-decoration:none;background-color:#0f0f0f;color:#fff}.btn,input[type="submit"].btn{display:inline-block;vertical-align:top;font-family:"Helvetica Neue",Helvetica,Tahoma,sans-serif;font-size:1em;font-weight:400;line-height:1.65em;text-align:center;text-decoration:none;color:#222;-webkit-appearance:none;outline:0;margin:0;border:0;border-radius:2px;box-shadow:none;cursor:pointer;background:#e0e3e5;padding:.55em 2.5em}.btn:hover,input[type="submit"].btn:hover{color:rgba(0,0,0,0.5);background:#b3b6b7}.btn::-moz-focus-inner{border:0;padding:0}.btn-big,input[type="submit"].btn-big{font-size:18px}.btn-small,input[type="submit"].btn-small{font-size:12.75px}.btn-smaller,input[type="submit"].btn-smaller{font-size:11.25px;vertical-align:baseline}.btn-round{border-radius:15px}.btn-outline,input[type="submit"].btn-outline{background:0;padding:.48750000000000004em 2.5em;border:1px solid #0f0f0f}.btn-outline:hover,input[type="submit"].btn-outline:hover{border-color:#b3b6b7}.btn-outline.btn-active{padding:.55em 2.5em}.btn-outline.bold{border-width:2px;padding:.42500000000000004em 2.5em}.btn-active,.btn[disabled],.btn-disabled{background:0;background:#b3b6b7;color:rgba(0,0,0,0.5)}.btn-active:hover,.btn[disabled]:hover,.btn-disabled:hover{color:rgba(0,0,0,0.5);background:#b3b6b7}.btn-active{box-shadow:0 1px 3px rgba(0,0,0,0.4) inset}.btn-outline.btn[disabled],.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(0,0,0,0.3);border:1px solid rgba(0,0,0,0.1)}.btn-outline.btn-active{background:0;color:rgba(0,0,0,0.4);border:0;box-shadow:0 1px 3px rgba(0,0,0,0.2) inset}.btn[disabled],.btn-disabled{cursor:default;box-shadow:none}.btn-blue,input[type="submit"].btn-blue{color:rgba(255,255,255,0.9);background:#2575ed}.btn-blue:hover,input[type="submit"].btn-blue:hover{color:rgba(255,255,255,0.6);background:#1a52a5}.btn-blue.btn-active,input[type="submit"].btn-blue.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-blue.btn-active,input[type="submit"].btn-blue.btn-active,.btn-blue.btn-disabled,input[type="submit"].btn-blue.btn-disabled,.btn-blue.btn[disabled],input[type="submit"].btn-blue.btn[disabled]{color:rgba(255,255,255,0.5);background:#1a52a5}.btn-blue.btn-active:hover,input[type="submit"].btn-blue.btn-active:hover,.btn-blue.btn-disabled:hover,input[type="submit"].btn-blue.btn-disabled:hover,.btn-blue.btn[disabled]:hover,input[type="submit"].btn-blue.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-blue.btn-outline,input[type="submit"].btn-blue.btn-outline{background:0;border-color:#2575ed;color:#2575ed}.btn-blue.btn-outline:hover,input[type="submit"].btn-blue.btn-outline:hover{color:rgba(255,255,255,0.9);background:#2575ed}.btn-blue.btn-outline.btn[disabled],input[type="submit"].btn-blue.btn-outline.btn[disabled],.btn-blue.btn-outline.btn-disabled,input[type="submit"].btn-blue.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(37,117,237,0.4);border:1px solid rgba(37,117,237,0.3)}.btn-blue.btn-outline.btn-active,input[type="submit"].btn-blue.btn-outline.btn-active{background:0;color:rgba(37,117,237,0.6);border:0;box-shadow:0 1px 3px rgba(26,82,165,0.6) inset}.btn-blue:hover,input[type="submit"].btn-blue:hover{color:rgba(255,255,255,0.6);background:#1a52a5}.btn-blue.btn-active,input[type="submit"].btn-blue.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-blue.btn-active,input[type="submit"].btn-blue.btn-active,.btn-blue.btn-disabled,input[type="submit"].btn-blue.btn-disabled,.btn-blue.btn[disabled],input[type="submit"].btn-blue.btn[disabled]{color:rgba(255,255,255,0.5);background:#1a52a5}.btn-blue.btn-active:hover,input[type="submit"].btn-blue.btn-active:hover,.btn-blue.btn-disabled:hover,input[type="submit"].btn-blue.btn-disabled:hover,.btn-blue.btn[disabled]:hover,input[type="submit"].btn-blue.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-blue.btn-outline,input[type="submit"].btn-blue.btn-outline{background:0;border-color:#2575ed;color:#2575ed}.btn-blue.btn-outline:hover,input[type="submit"].btn-blue.btn-outline:hover{color:rgba(255,255,255,0.9);background:#2575ed}.btn-blue.btn-outline.btn[disabled],input[type="submit"].btn-blue.btn-outline.btn[disabled],.btn-blue.btn-outline.btn-disabled,input[type="submit"].btn-blue.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(37,117,237,0.4);border:1px solid rgba(37,117,237,0.3)}.btn-blue.btn-outline.btn-active,input[type="submit"].btn-blue.btn-outline.btn-active{background:0;color:rgba(37,117,237,0.6);border:0;box-shadow:0 1px 3px rgba(26,82,165,0.6) inset}.btn-red,input[type="submit"].btn-red{color:rgba(255,255,255,0.9);background:#de2c3b}.btn-red:hover,input[type="submit"].btn-red:hover{color:rgba(255,255,255,0.6);background:#b2232f}.btn-red.btn-active,input[type="submit"].btn-red.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-red.btn-active,input[type="submit"].btn-red.btn-active,.btn-red.btn-disabled,input[type="submit"].btn-red.btn-disabled,.btn-red.btn[disabled],input[type="submit"].btn-red.btn[disabled]{color:rgba(255,255,255,0.5);background:#b2232f}.btn-red.btn-active:hover,input[type="submit"].btn-red.btn-active:hover,.btn-red.btn-disabled:hover,input[type="submit"].btn-red.btn-disabled:hover,.btn-red.btn[disabled]:hover,input[type="submit"].btn-red.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-red.btn-outline,input[type="submit"].btn-red.btn-outline{background:0;border-color:#de2c3b;color:#de2c3b}.btn-red.btn-outline:hover,input[type="submit"].btn-red.btn-outline:hover{color:rgba(255,255,255,0.9);background:#de2c3b}.btn-red.btn-outline.btn[disabled],input[type="submit"].btn-red.btn-outline.btn[disabled],.btn-red.btn-outline.btn-disabled,input[type="submit"].btn-red.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(222,44,59,0.4);border:1px solid rgba(222,44,59,0.3)}.btn-red.btn-outline.btn-active,input[type="submit"].btn-red.btn-outline.btn-active{background:0;color:rgba(222,44,59,0.6);border:0;box-shadow:0 1px 3px rgba(178,35,47,0.6) inset}.btn-red:hover,input[type="submit"].btn-red:hover{color:rgba(255,255,255,0.6);background:#b2232f}.btn-red.btn-active,input[type="submit"].btn-red.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-red.btn-active,input[type="submit"].btn-red.btn-active,.btn-red.btn-disabled,input[type="submit"].btn-red.btn-disabled,.btn-red.btn[disabled],input[type="submit"].btn-red.btn[disabled]{color:rgba(255,255,255,0.5);background:#b2232f}.btn-red.btn-active:hover,input[type="submit"].btn-red.btn-active:hover,.btn-red.btn-disabled:hover,input[type="submit"].btn-red.btn-disabled:hover,.btn-red.btn[disabled]:hover,input[type="submit"].btn-red.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-red.btn-outline,input[type="submit"].btn-red.btn-outline{background:0;border-color:#de2c3b;color:#de2c3b}.btn-red.btn-outline:hover,input[type="submit"].btn-red.btn-outline:hover{color:rgba(255,255,255,0.9);background:#de2c3b}.btn-red.btn-outline.btn[disabled],input[type="submit"].btn-red.btn-outline.btn[disabled],.btn-red.btn-outline.btn-disabled,input[type="submit"].btn-red.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(222,44,59,0.4);border:1px solid rgba(222,44,59,0.3)}.btn-red.btn-outline.btn-active,input[type="submit"].btn-red.btn-outline.btn-active{background:0;color:rgba(222,44,59,0.6);border:0;box-shadow:0 1px 3px rgba(178,35,47,0.6) inset}.btn-green,input[type="submit"].btn-green{color:rgba(255,255,255,0.9);background:#2c9f42}.btn-green:hover,input[type="submit"].btn-green:hover{color:rgba(255,255,255,0.6);background:#237f35}.btn-green.btn-active,input[type="submit"].btn-green.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-green.btn-active,input[type="submit"].btn-green.btn-active,.btn-green.btn-disabled,input[type="submit"].btn-green.btn-disabled,.btn-green.btn[disabled],input[type="submit"].btn-green.btn[disabled]{color:rgba(255,255,255,0.5);background:#237f35}.btn-green.btn-active:hover,input[type="submit"].btn-green.btn-active:hover,.btn-green.btn-disabled:hover,input[type="submit"].btn-green.btn-disabled:hover,.btn-green.btn[disabled]:hover,input[type="submit"].btn-green.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-green.btn-outline,input[type="submit"].btn-green.btn-outline{background:0;border-color:#2c9f42;color:#2c9f42}.btn-green.btn-outline:hover,input[type="submit"].btn-green.btn-outline:hover{color:rgba(255,255,255,0.9);background:#2c9f42}.btn-green.btn-outline.btn[disabled],input[type="submit"].btn-green.btn-outline.btn[disabled],.btn-green.btn-outline.btn-disabled,input[type="submit"].btn-green.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(44,159,66,0.4);border:1px solid rgba(44,159,66,0.3)}.btn-green.btn-outline.btn-active,input[type="submit"].btn-green.btn-outline.btn-active{background:0;color:rgba(44,159,66,0.6);border:0;box-shadow:0 1px 3px rgba(35,127,53,0.6) inset}.btn-green:hover,input[type="submit"].btn-green:hover{color:rgba(255,255,255,0.6);background:#237f35}.btn-green.btn-active,input[type="submit"].btn-green.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-green.btn-active,input[type="submit"].btn-green.btn-active,.btn-green.btn-disabled,input[type="submit"].btn-green.btn-disabled,.btn-green.btn[disabled],input[type="submit"].btn-green.btn[disabled]{color:rgba(255,255,255,0.5);background:#237f35}.btn-green.btn-active:hover,input[type="submit"].btn-green.btn-active:hover,.btn-green.btn-disabled:hover,input[type="submit"].btn-green.btn-disabled:hover,.btn-green.btn[disabled]:hover,input[type="submit"].btn-green.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-green.btn-outline,input[type="submit"].btn-green.btn-outline{background:0;border-color:#2c9f42;color:#2c9f42}.btn-green.btn-outline:hover,input[type="submit"].btn-green.btn-outline:hover{color:rgba(255,255,255,0.9);background:#2c9f42}.btn-green.btn-outline.btn[disabled],input[type="submit"].btn-green.btn-outline.btn[disabled],.btn-green.btn-outline.btn-disabled,input[type="submit"].btn-green.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(44,159,66,0.4);border:1px solid rgba(44,159,66,0.3)}.btn-green.btn-outline.btn-active,input[type="submit"].btn-green.btn-outline.btn-active{background:0;color:rgba(44,159,66,0.6);border:0;box-shadow:0 1px 3px rgba(35,127,53,0.6) inset}.btn-black,input[type="submit"].btn-black{color:rgba(255,255,255,0.9);background:#0f0f0f}.btn-black:hover,input[type="submit"].btn-black:hover{color:rgba(255,255,255,0.6);background:#363738}.btn-black.btn-active,input[type="submit"].btn-black.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.7) inset}.btn-black.btn-active,input[type="submit"].btn-black.btn-active,.btn-black.btn-disabled,input[type="submit"].btn-black.btn-disabled,.btn-black.btn[disabled],input[type="submit"].btn-black.btn[disabled]{color:rgba(255,255,255,0.5);background:#363738}.btn-black.btn-active:hover,input[type="submit"].btn-black.btn-active:hover,.btn-black.btn-disabled:hover,input[type="submit"].btn-black.btn-disabled:hover,.btn-black.btn[disabled]:hover,input[type="submit"].btn-black.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-black.btn-outline,input[type="submit"].btn-black.btn-outline{background:0;border-color:#0f0f0f;color:#0f0f0f}.btn-black.btn-outline:hover,input[type="submit"].btn-black.btn-outline:hover{color:rgba(255,255,255,0.9);background:#0f0f0f}.btn-black.btn-outline.btn[disabled],input[type="submit"].btn-black.btn-outline.btn[disabled],.btn-black.btn-outline.btn-disabled,input[type="submit"].btn-black.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(15,15,15,0.4);border:1px solid rgba(15,15,15,0.3)}.btn-black.btn-outline.btn-active,input[type="submit"].btn-black.btn-outline.btn-active{background:0;color:rgba(15,15,15,0.6);border:0;box-shadow:0 1px 3px rgba(54,55,56,0.6) inset}.btn-black:hover,input[type="submit"].btn-black:hover{color:rgba(255,255,255,0.6);background:#363738}.btn-black.btn-active,input[type="submit"].btn-black.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.7) inset}.btn-black.btn-active,input[type="submit"].btn-black.btn-active,.btn-black.btn-disabled,input[type="submit"].btn-black.btn-disabled,.btn-black.btn[disabled],input[type="submit"].btn-black.btn[disabled]{color:rgba(255,255,255,0.5);background:#363738}.btn-black.btn-active:hover,input[type="submit"].btn-black.btn-active:hover,.btn-black.btn-disabled:hover,input[type="submit"].btn-black.btn-disabled:hover,.btn-black.btn[disabled]:hover,input[type="submit"].btn-black.btn[disabled]:hover{color:rgba(255,255,255,0.5)}.btn-black.btn-outline,input[type="submit"].btn-black.btn-outline{background:0;border-color:#0f0f0f;color:#0f0f0f}.btn-black.btn-outline:hover,input[type="submit"].btn-black.btn-outline:hover{color:rgba(255,255,255,0.9);background:#0f0f0f}.btn-black.btn-outline.btn[disabled],input[type="submit"].btn-black.btn-outline.btn[disabled],.btn-black.btn-outline.btn-disabled,input[type="submit"].btn-black.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(15,15,15,0.4);border:1px solid rgba(15,15,15,0.3)}.btn-black.btn-outline.btn-active,input[type="submit"].btn-black.btn-outline.btn-active{background:0;color:rgba(15,15,15,0.6);border:0;box-shadow:0 1px 3px rgba(54,55,56,0.6) inset}.btn-yellow,input[type="submit"].btn-yellow{color:rgba(0,0,0,0.9);background:#ffc800}.btn-yellow:hover,input[type="submit"].btn-yellow:hover{color:rgba(0,0,0,0.6);background:#cca000}.btn-yellow.btn-active,input[type="submit"].btn-yellow.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-yellow.btn-active,input[type="submit"].btn-yellow.btn-active,.btn-yellow.btn-disabled,input[type="submit"].btn-yellow.btn-disabled,.btn-yellow.btn[disabled],input[type="submit"].btn-yellow.btn[disabled]{color:rgba(0,0,0,0.5);background:#cca000}.btn-yellow.btn-active:hover,input[type="submit"].btn-yellow.btn-active:hover,.btn-yellow.btn-disabled:hover,input[type="submit"].btn-yellow.btn-disabled:hover,.btn-yellow.btn[disabled]:hover,input[type="submit"].btn-yellow.btn[disabled]:hover{color:rgba(0,0,0,0.5)}.btn-yellow.btn-outline,input[type="submit"].btn-yellow.btn-outline{background:0;border-color:#ffc800;color:#ffc800}.btn-yellow.btn-outline:hover,input[type="submit"].btn-yellow.btn-outline:hover{color:rgba(0,0,0,0.9);background:#ffc800}.btn-yellow.btn-outline.btn[disabled],input[type="submit"].btn-yellow.btn-outline.btn[disabled],.btn-yellow.btn-outline.btn-disabled,input[type="submit"].btn-yellow.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(255,200,0,0.4);border:1px solid rgba(255,200,0,0.3)}.btn-yellow.btn-outline.btn-active,input[type="submit"].btn-yellow.btn-outline.btn-active{background:0;color:rgba(255,200,0,0.6);border:0;box-shadow:0 1px 3px rgba(204,160,0,0.6) inset}.btn-yellow:hover,input[type="submit"].btn-yellow:hover{color:rgba(0,0,0,0.6);background:#cca000}.btn-yellow.btn-active,input[type="submit"].btn-yellow.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.5) inset}.btn-yellow.btn-active,input[type="submit"].btn-yellow.btn-active,.btn-yellow.btn-disabled,input[type="submit"].btn-yellow.btn-disabled,.btn-yellow.btn[disabled],input[type="submit"].btn-yellow.btn[disabled]{color:rgba(0,0,0,0.5);background:#cca000}.btn-yellow.btn-active:hover,input[type="submit"].btn-yellow.btn-active:hover,.btn-yellow.btn-disabled:hover,input[type="submit"].btn-yellow.btn-disabled:hover,.btn-yellow.btn[disabled]:hover,input[type="submit"].btn-yellow.btn[disabled]:hover{color:rgba(0,0,0,0.5)}.btn-yellow.btn-outline,input[type="submit"].btn-yellow.btn-outline{background:0;border-color:#ffc800;color:#ffc800}.btn-yellow.btn-outline:hover,input[type="submit"].btn-yellow.btn-outline:hover{color:rgba(0,0,0,0.9);background:#ffc800}.btn-yellow.btn-outline.btn[disabled],input[type="submit"].btn-yellow.btn-outline.btn[disabled],.btn-yellow.btn-outline.btn-disabled,input[type="submit"].btn-yellow.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(255,200,0,0.4);border:1px solid rgba(255,200,0,0.3)}.btn-yellow.btn-outline.btn-active,input[type="submit"].btn-yellow.btn-outline.btn-active{background:0;color:rgba(255,200,0,0.6);border:0;box-shadow:0 1px 3px rgba(204,160,0,0.6) inset}.btn-white,input[type="submit"].btn-white{color:rgba(0,0,0,0.9);background:#fff}.btn-white:hover,input[type="submit"].btn-white:hover{color:rgba(0,0,0,0.6);background:#ededed}.btn-white.btn-active,input[type="submit"].btn-white.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.2) inset}.btn-white.btn-active,input[type="submit"].btn-white.btn-active,.btn-white.btn-disabled,input[type="submit"].btn-white.btn-disabled,.btn-white.btn[disabled],input[type="submit"].btn-white.btn[disabled]{color:rgba(0,0,0,0.5);background:#ededed}.btn-white.btn-active:hover,input[type="submit"].btn-white.btn-active:hover,.btn-white.btn-disabled:hover,input[type="submit"].btn-white.btn-disabled:hover,.btn-white.btn[disabled]:hover,input[type="submit"].btn-white.btn[disabled]:hover{color:rgba(0,0,0,0.5)}.btn-white.btn-outline,input[type="submit"].btn-white.btn-outline{background:0;border-color:#fff;color:#fff}.btn-white.btn-outline:hover,input[type="submit"].btn-white.btn-outline:hover{color:rgba(0,0,0,0.9);background:#fff}.btn-white.btn-outline.btn[disabled],input[type="submit"].btn-white.btn-outline.btn[disabled],.btn-white.btn-outline.btn-disabled,input[type="submit"].btn-white.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(255,255,255,0.4);border:1px solid rgba(255,255,255,0.3)}.btn-white.btn-outline.btn-active,input[type="submit"].btn-white.btn-outline.btn-active{background:0;color:rgba(255,255,255,0.6);border:0;box-shadow:0 1px 3px rgba(237,237,237,0.6) inset}.btn-white:hover,input[type="submit"].btn-white:hover{color:rgba(0,0,0,0.6);background:#ededed}.btn-white.btn-active,input[type="submit"].btn-white.btn-active{box-shadow:0 1px 4px rgba(0,0,0,0.2) inset}.btn-white.btn-active,input[type="submit"].btn-white.btn-active,.btn-white.btn-disabled,input[type="submit"].btn-white.btn-disabled,.btn-white.btn[disabled],input[type="submit"].btn-white.btn[disabled]{color:rgba(0,0,0,0.5);background:#ededed}.btn-white.btn-active:hover,input[type="submit"].btn-white.btn-active:hover,.btn-white.btn-disabled:hover,input[type="submit"].btn-white.btn-disabled:hover,.btn-white.btn[disabled]:hover,input[type="submit"].btn-white.btn[disabled]:hover{color:rgba(0,0,0,0.5)}.btn-white.btn-outline,input[type="submit"].btn-white.btn-outline{background:0;border-color:#fff;color:#fff}.btn-white.btn-outline:hover,input[type="submit"].btn-white.btn-outline:hover{color:rgba(0,0,0,0.9);background:#fff}.btn-white.btn-outline.btn[disabled],input[type="submit"].btn-white.btn-outline.btn[disabled],.btn-white.btn-outline.btn-disabled,input[type="submit"].btn-white.btn-outline.btn-disabled{background:0;box-shadow:none;color:rgba(255,255,255,0.4);border:1px solid rgba(255,255,255,0.3)}.btn-white.btn-outline.btn-active,input[type="submit"].btn-white.btn-outline.btn-active{background:0;color:rgba(255,255,255,0.6);border:0;box-shadow:0 1px 3px rgba(237,237,237,0.6) inset}.btn-white.btn-outline.btn-active{box-shadow:none;border:1px solid rgba(255,255,255,0.3);padding:.48750000000000004em 2.5em}.btn-single,.btn-group{display:inline-block;margin-right:2px;vertical-align:bottom}.btn-single:after,.btn-group:after{content:"";display:table;clear:both}.btn-single:after,.btn-group:after{content:"";display:table;clear:both}.btn-single>.btn,.btn-single>input,.btn-group>.btn,.btn-group>input{float:left;border-radius:0;margin-left:-1px}.btn-single>.btn{border-radius:2px}.btn-group>.btn:first-child{border-radius:2px 0 0 2px}.btn-group>.btn:last-child{border-radius:0 2px 2px 0}.btn-group>.btn.btn-round:first-child,.btn-group>.input-search:first-child{border-radius:15px 0 0 15px}.btn-group>.btn.btn-round:last-child,.btn-group>.input-search:last-child{border-radius:0 15px 15px 0}.tools-alert{padding:12px 15px;background:#f7f8f8;color:#0f0f0f;margin-bottom:1.65em}.tools-message{display:none;position:fixed;z-index:100;top:10px;right:10px;max-width:350px;line-height:1.5;font-size:95%;padding:12px 15px;color:#0f0f0f;background:#e0e3e5}.tools-message ul{margin:0;list-style:none}.tools-message-black,.tools-message-blue,.tools-message-red,.tools-message-green{color:rgba(255,255,255,0.95)}.tools-message-black{background:#0f0f0f}.tools-message-blue{background:#2575ed}.tools-message-red{background:#de2c3b}.tools-message-yellow{background:#ffc800}.tools-message-green{background:#2c9f42}.tools-alert-black{background:#dadada}.tools-alert-blue{background:#d3e3fb}.tools-alert-red{background:#f8d5d8}.tools-alert-yellow{background:#fff4cc}.tools-alert-green{background:#d5ecd9}.label,.badge{background:#e0e3e5;font-size:12.75px;display:inline-block;line-height:1;padding:4px 7px 3px 7px;color:#0f0f0f;text-align:center;font-weight:normal;text-transform:uppercase}.label-outline{background:0;border:1px solid #0f0f0f;padding:3px 6px 2px 6px}.badge{border-radius:15px}.badge-small{font-size:11.25px;padding:3px 5px}.label-black,.label-blue,.label-red,.label-green,.badge-black,.badge-blue,.badge-red,.badge-green{color:#fff}.label-black,.badge-black{background:#0f0f0f}.label-blue,.badge-blue{background:#2575ed}.label-red,.badge-red{background:#de2c3b}.label-green,.badge-green{background:#2c9f42}.label-yellow,.badge-yellow{background:#ffc800}.label-white,.badge-white{background:#fff}.label-black.label-outline,.label-blue.label-outline,.label-red.label-outline,.label-green.label-outline,.label-yellow.label-outline,.label-white.label-outline{background:0;color:#0f0f0f}.label-blue.label-outline{border-color:#2575ed;color:#2575ed}.label-red.label-outline{border-color:#de2c3b;color:#de2c3b}.label-green.label-outline{border-color:#2c9f42;color:#2c9f42}.label-yellow.label-outline{border-color:#ffc800;color:#ffc800}.label-white.label-outline{border-color:#fff;color:#fff}#tools-progress{position:fixed;top:0;left:0;width:100%;z-index:1000000;height:10px}#tools-progress span{display:block;width:100%;height:100%;background-color:#1a52a5;background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,0.2) 25%,transparent 25%,transparent 50%,rgba(255,255,255,0.2) 50%,rgba(255,255,255,0.2) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,0.2) 25%,transparent 25%,transparent 50%,rgba(255,255,255,0.2) 50%,rgba(255,255,255,0.2) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,0.2) 25%,transparent 25%,transparent 50%,rgba(255,255,255,0.2) 50%,rgba(255,255,255,0.2) 75%,transparent 75%,transparent);-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite;background-size:40px 40px}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.accordion-title{position:relative;display:block;margin:0;margin-bottom:2px;padding:12px 50px 12px 18px;color:#000;text-decoration:none;font-weight:normal;font-size:.9375em;line-height:1,4em;background:#f7f8f8;cursor:pointer;-webkit-touch-callout:none;-webkit-user-select:none;-khtml-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none}.accordion-title.accordion-title-opened{margin-bottom:0;font-weight:bold;background:#f0f1f2}.accordion-title a,.accordion-title:hover{color:#000;text-decoration:none}.accordion-panel{padding:1.65em;margin-bottom:10px}.accordion-toggle{position:absolute;top:50%;margin-top:-8px;right:20px;padding:0;font-size:0;line-height:1}.accordion-toggle-closed{background:#000;width:1px;height:15px;margin-left:1px}.accordion-toggle-closed:before{position:absolute;top:7px;left:-7px;content:"";width:15px;height:1px;background:#000}.accordion-toggle-opened:before{position:absolute;top:7px;left:-8px;content:"";width:15px;height:1px;background:#000}.filterbox{position:relative}.filterbox input{padding-right:30px}.filterbox span{position:absolute;z-index:2;top:0;right:1px;width:26px;height:100%;cursor:pointer}.filterbox span:after{content:"";display:inline-block;position:relative;top:50%;margin-left:8px;margin-top:-21px;width:0;height:0;vertical-align:middle;border-top:5px solid rgba(0,0,0,0.6);border-right:5px solid transparent;border-left:5px solid transparent}.filterbox-list{z-index:1000;position:absolute;left:0;display:none;margin:0;list-style:none;background:#fff;width:100%;box-shadow:0 1px 3px rgba(0,0,0,0.2);max-height:250px;overflow:auto}.filterbox-list li{padding:4px 10px;color:#000;cursor:pointer}.filterbox-list li:hover{background:#f0f1f2}.filterbox-list li.active{background:#2575ed;color:#fff}.tooltip{position:absolute;z-index:10000;display:inline-block;color:#fff;padding:2px 10px;font-size:12.75px;line-height:1.5em;max-width:250px;background:#0f0f0f}.tooltip-theme-red{background:#de2c3b}.tooltip-theme-blue{background:#2575ed}.tooltip-theme-green{background:#2c9f42}.tooltip-theme-yellow{background:#ffc800}.tooltip-theme-white{background:#fff}.tooltip-theme-yellow,.tooltip-theme-white{color:#000}.dropdown{display:none;position:absolute;z-index:102;top:0;right:0;width:250px;color:#0f0f0f;background:#fff;box-shadow:0 1px 5px rgba(0,0,0,0.3);overflow:auto}.dropdown section{max-height:250px;overflow:auto;padding:20px}.dropdown footer{padding:20px}ul.dropdown{max-height:300px;list-style:none;margin:0;line-height:1.5;font-size:95%;padding:0}ul.dropdown a{display:block;padding:7px 15px;text-decoration:none;color:#0f0f0f}ul.dropdown a:hover{background:#eee}ul.dropdown li.divider{border-bottom:1px solid #e2e2e2}.caret{display:inline-block;width:0;height:0;margin-left:.3em;vertical-align:middle;border-top:5px solid;border-right:5px solid transparent;border-left:5px solid transparent}.caret.caret-up{border-top:0;border-bottom:4px solid}.livesearch-box{position:relative;display:inline-block;width:100%}.livesearch-box input{padding-left:30px}.livesearch-box .close{position:absolute;top:.2em;right:5px;z-index:2;padding:4px 6px;line-height:1;font-size:20px;cursor:pointer;color:#000;text-decoration:none;filter:alpha(opacity=50);-moz-opacity:.5;opacity:.5}.livesearch-box .close:before{content:'\00D7'}.livesearch-box .close:hover{filter:alpha(opacity=100);-moz-opacity:1;opacity:1}.livesearch-box .close:before{content:'\00D7'}.livesearch-box .close:hover{filter:alpha(opacity=100);-moz-opacity:1;opacity:1}.livesearch-icon{position:absolute;top:53%;left:10px;z-index:2}.livesearch-icon:before,.livesearch-icon:after{content:"";position:absolute;top:50%;left:0;margin:-8px 0 0;background:rgba(0,0,0,0.5)}.livesearch-icon:before{width:10px;height:10px;border:2px solid rgba(0,0,0,0.5);background:transparent;border-radius:12px}.livesearch-icon:after{left:10px;width:2px;height:7px;margin-top:0;-webkit-transform:rotate(-45deg);-moz-transform:rotate(-45deg);-ms-transform:rotate(-45deg);-o-transform:rotate(-45deg);transform:rotate(-45deg)}.tools-droparea{position:relative;overflow:hidden;padding:80px 20px;border:3px dashed rgba(0,0,0,0.1)}.tools-droparea.drag-hover{background:rgba(200,222,250,0.75)}.tools-droparea.drag-drop{background:rgba(250,248,200,0.5)}.tools-droparea-placeholder{text-align:center;font-size:11px;color:rgba(0,0,0,0.5)}.autocomplete{position:absolute;z-index:1000;left:0;display:none;margin:0;list-style:none;background:#fff;width:250px;box-shadow:0 1px 3px rgba(0,0,0,0.2);max-height:250px;overflow:auto}.autocomplete a{padding:4px 10px;color:#000;display:block;text-decoration:none}.autocomplete a:hover{background:#f0f1f2}.autocomplete a.active{background:#2575ed;color:#fff}#modal-overlay{position:fixed;top:0;left:0;margin:auto;overflow:auto;width:100%;height:100%;background-color:#000!important;filter:alpha(opacity=30);-moz-opacity:.3;opacity:.3;z-index:100}.modal-blur{-webkit-filter:blur(3px);-moz-filter:blur(3px);-ms-filter:blur(3px);filter:blur(3px)}.modal-box{position:fixed;top:0;left:0;bottom:0;right:0;overflow-x:hidden;overflow-y:auto;z-index:101}.modal{position:relative;margin:auto;margin-bottom:20px;padding:0;background:#fff;color:#000;box-shadow:0 1px 70px rgba(0,0,0,0.5)}.modal header{padding:30px 40px 5px 40px;font-size:18px;font-weight:bold}.modal section{padding:30px 40px 50px 40px}.modal footer button{width:100%;border-radius:0}.modal-close{position:absolute;top:8px;right:12px;width:30px;height:30px;text-align:right;color:#bbb;font-size:30px;font-weight:300;cursor:pointer}.modal-close:hover{color:#000}.group:after{content:"";display:table;clear:both}.group:after{content:"";display:table;clear:both}.hide{display:none}.highlight{background-color:#f7f3e2}.big{font-size:18px}.small{font-size:12.75px}.smaller{font-size:11.25px}.nowrap,.nowrap td{white-space:nowrap}.req,.required{font-weight:normal;color:#de2c3b}.error{color:#de2c3b}.success{color:#2c9f42}.text-centered{text-align:center}.text-right{text-align:right}.last{margin-right:0!important}.pause{margin-bottom:.825em!important}.end{margin-bottom:0!important}.normal{font-weight:normal}.light{font-weight:300}.bold{font-weight:bold}.italic{font-style:italic}.left{float:left}.right{float:right}.upper{text-transform:uppercase}.list-flat{margin-left:0;list-style:none}.color-black{color:#0f0f0f}.color-white{color:#fff}.color-gray-10{color:rgba(0,0,0,0.1)}.color-gray-20{color:rgba(0,0,0,0.2)}.color-gray-30{color:rgba(0,0,0,0.3)}.color-gray-40{color:rgba(0,0,0,0.4)}.color-gray-50{color:rgba(0,0,0,0.5)}.color-gray-60{color:rgba(0,0,0,0.6)}.color-gray-70{color:rgba(0,0,0,0.7)}.color-gray-80{color:rgba(0,0,0,0.8)}.color-gray-90{color:rgba(0,0,0,0.9)}.color-white-10{color:rgba(255,255,255,0.1)}.color-white-20{color:rgba(255,255,255,0.2)}.color-white-30{color:rgba(255,255,255,0.3)}.color-white-40{color:rgba(255,255,255,0.4)}.color-white-50{color:rgba(255,255,255,0.5)}.color-white-60{color:rgba(255,255,255,0.6)}.color-white-70{color:rgba(255,255,255,0.7)}.color-white-80{color:rgba(255,255,255,0.8)}.color-white-90{color:rgba(255,255,255,0.9)}.video-wrapper{height:0;padding-bottom:56.25%;position:relative;margin-bottom:1.65em}.video-wrapper iframe,.video-wrapper object,.video-wrapper embed{position:absolute;top:0;left:0;width:100%;height:100%}@media only screen and (max-width:767px){.left,.right{float:none}.hide-on-mobile{display:none}}.str{color:#d14}.kwd{color:#333}.com{color:#998}.typ{color:#458}.lit{color:#458}.pun{color:#888}.opn{color:#333}.clo{color:#333}.tag{color:#367ac3}.atn{color:#51a7c9}.atv{color:#709c1a}.dec{color:#666}.var{color:teal}.fun{color:#900}.linenums ol li{list-style-type:none;counter-increment:list;position:relative}.linenums ol li:after{content:counter(list);position:absolute;left:-3.3em;border-right:1px solid #e5e5e5;padding-right:9px;width:2.45em;text-align:right;color:rgba(0,0,0,0.3);font-size:12px}@media only screen and (max-width:767px){.mobile-width-100{width:100%}.units-row .unit-90,.units-row .unit-80,.units-row .unit-75,.units-row .unit-70,.units-row .unit-66,.units-row .unit-65,.units-row .unit-60,.units-row .unit-50,.units-row .unit-40,.units-row .unit-35,.units-row .unit-33,.units-row .unit-30,.units-row .unit-25,.units-row .unit-20,.units-row .unit-10{width:100%;float:none;margin-left:0;margin-bottom:1.65em}.unit-push-90,.unit-push-80,.unit-push-75,.unit-push-70,.unit-push-66,.unit-push-65,.unit-push-60,.unit-push-50,.unit-push-40,.unit-push-35,.unit-push-33,.unit-push-30,.unit-push-25,.unit-push-20,.unit-push-10{left:0}.units-row .unit-push-right{float:none}.units-mobile-50 .unit-90,.units-mobile-50 .unit-80,.units-mobile-50 .unit-75,.units-mobile-50 .unit-70,.units-mobile-50 .unit-66,.units-mobile-50 .unit-65,.units-mobile-50 .unit-60,.units-mobile-50 .unit-40,.units-mobile-50 .unit-30,.units-mobile-50 .unit-35,.units-mobile-50 .unit-33,.units-mobile-50 .unit-25,.units-mobile-50 .unit-20,.units-mobile-50 .unit-10{float:left;margin-left:3%;width:48.5%}.units-mobile-50 .unit-90:first-child,.units-mobile-50 .unit-80:first-child,.units-mobile-50 .unit-75:first-child,.units-mobile-50 .unit-70:first-child,.units-mobile-50 .unit-66:first-child,.units-mobile-50 .unit-65:first-child,.units-mobile-50 .unit-60:first-child,.units-mobile-50 .unit-40:first-child,.units-mobile-50 .unit-35:first-child,.units-mobile-50 .unit-30:first-child,.units-mobile-50 .unit-33:first-child,.units-mobile-50 .unit-25:first-child,.units-mobile-50 .unit-20:first-child,.units-mobile-50 .unit-10:first-child{margin-left:0}}@media only screen and (max-width:767px){.blocks-2,.blocks-3,.blocks-4,.blocks-5,.blocks-6{margin-left:0;margin-bottom:1.65em}.blocks-2>li,.blocks-3>li,.blocks-4>li,.blocks-5>li,.blocks-6>li{float:none;margin-left:0;width:100%}.blocks-mobile-50>li,.blocks-mobile-33>li{float:left;margin-left:3%}.blocks-mobile-33,.blocks-mobile-50{margin-left:-3%}.blocks-mobile-50>li{width:47%}.blocks-mobile-33>li{width:30.333333333333332%}}@media(min-width:768px) and (max-width:979px){h1{font-size:2.25em;line-height:1.125}h2{font-size:1.5em;line-height:1.25}h3{font-size:1.3125em;line-height:1.25}h4{font-size:1.125em;line-height:1.22222222}h5{font-size:1em}h6{font-size:.75em}}@media(max-width:767px){h1{font-size:2.25em;line-height:1.25}h2{font-size:1.5em;line-height:1.15384615}h3{font-size:1.3125em;line-height:1.13636364}h4{font-size:1.125em;line-height:1.11111111}h5{font-size:1em}h6{font-size:.75em}.lead{font-size:1.2em}ul,ol,ul ul,ol ol,ul ol,ol ul{margin-left:1.65em}blockquote{margin-left:0}}@media only screen and (max-width:767px){.navbar.navbar-left,.navbar.navbar-right,.navbar li,.navbar.navbar-left li,.navbar.navbar-right li{float:none;text-align:left;width:auto}.navbar li,.navbar.navbar-right li{margin-left:0;margin-right:0}.fullwidth ul,.fullwidth li{width:auto}.fullwidth li{display:block}}@media only screen and (max-width:767px){.forms-list label{display:inline-block}}@media print{*{text-shadow:none!important;color:#000!important;background:transparent!important;box-shadow:none!important;font-size:12pt}.h1,h1{font-size:36pt}.h2,h2{font-size:24pt}.h3,h3{font-size:18pt}.h4,h4{font-size:14pt}.h5,h5{font-size:12pt}.h6,h6{font-size:12pt}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="javascript:"]:after,a[href^="#"]:after{content:""}pre,blockquote{page-break-inside:avoid}blockquote{border:0;font-style:italic}img{max-width:100%!important}select{background:#fff!important}}
.agent-error {
    background-color: #FF6FCF;
    margin: 5px;
}
.agent-warning {
    background-color: #FFD732;
    margin: 5px;
}
.agent-info {
    background-color: #80FF00;
    margin: 5px;
}
.pad-5 {
    padding: 5px;
}
.margin-5 {
    margin: 5px;
}
.request {
    font-weight: bold;
    font-size: 20px;
    margin-left: 10px;
    padding: 5px;
}
.thread {
    font-style: italic;
    font-size: 18px;
    margin-left: 20px;
    margin-top: 10px;
}
.type-ts-msg {
    margin-left: 30px;
    font-size: 14px;
    padding: 5px;
}
.logdb-info-records {
    background-color: #E5E5E5;
    position: fixed;
    width: 700px;
    height: 500px;
    border-style: groove;
    border-width: 3px;
    overflow: scroll;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    z-index:999;
}
.shadow {
    -webkit-box-shadow: 10px 10px 20px #666666;
    -moz-box-shadow: 10px 10px 20px #666666;
    -o-box-shadow: 10px 10px 20px #666666;
    box-shadow: 10px 10px 20px #666666;
}
.hide {
    display: none;
}
.show {
    display: block;
}